src/cdomains/lval.ml

```diff
@@ -385,22 +385,90 @@ end
 module NormalLatRepr (Idx: IntDomain.Z) =
 struct
   include NormalLat (Idx)
+  module Offset = struct
```

Comment:
Can we maybe change this module name to something such as `ReprOffset` to make it clear that it is distinct from normal offsets?